addons/web_settings_dashboard/static/src/js/dashboard.js

```diff
@@ -104,15 +104,19 @@ var DashboardInvitations = Widget.extend({
             }
             emails = _.difference(emails, invalidEmails);
 
-            // filter out already processed or pending addresses
-            var pendingEmails = _.map(this.data.pending_users, function (info) {
-                return info[1];
-            });
-            var existingEmails = _.intersection(emails, this.emails.concat(pendingEmails));
-            if (existingEmails.length) {
-                this.do_warn(_.str.sprintf(_t('The following email addresses already exist: %s.'), existingEmails.join(', ')));
+            if(this.data.resend_invitation){
+                emails = _.difference(emails, invalidEmails);
+            } else {
```

Comment:
You don't need the else, as you change nothing in the if part, just invert the condition.
See this
```
if (!this.data.resend_invitation) {
    var pendingEmails = _.map(this.data.pending_users, function (info) {
        return info[1];
    });
    [...]
```